### DISCUSSION TITLE
Welcome to FoobarRemoteController2 Discussions!

### DISCUSSION THREAD

**Discussion by deckerth** [deckerth/FoobarRemoteController2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by deckerth** [comment]
We are working on a complete rehaul of the project. Here are some screenshots:
![Screenshot_2024-11-03-18-17-47-86_f4b92f21f7c7d41f27884173edec0663](https://github.com/user-attachments/assets/5d3a8de4-9b22-4a4d-99b2-cd87113a03be)
![Screenshot_2024-11-03-18-17-57-52_f4b92f21f7c7d41f27884173edec0663](https://github.com/user-attachments/assets/c8c3444e-a32e-41b4-8057-1d8ebdbd9507)
![Screenshot_2024-11-03-18-18-24-66_f4b92f21f7c7d41f27884173edec0663](https://github.com/user-attachments/assets/dda0085e-6c47-403a-a24f-c5178fee2f6a)
![Screenshot_2024-11-03-18-18-58-65_f4b92f21f7c7d41f27884173edec0663](https://github.com/user-attachments/assets/8293fe9d-7d38-4bab-af60-9e24ea04b62e)
![Screenshot_2024-11-03-18-19-02-18_f4b92f21f7c7d41f27884173edec0663](https://github.com/user-attachments/assets/c21624b0-da4a-4bdb-b9d5-4b39a0ca6591)
![Screenshot_2024-11-03-18-19-07-41_f4b92f21f7c7d41f27884173edec0663](https://github.com/user-attachments/assets/9cde10d3-e77c-4f4a-a2fc-5399af33902e)
![Screenshot_2024-11-03-18-19-35-73_f4b92f21f7c7d41f27884173edec0663](https://github.com/user-attachments/assets/599183ec-2325-42bc-8c4d-b1e6c7f23da7)

**Comment by eikaramba** [comment]
Greatings from Münster! I found this project after i saw that the famous foobar2000 controller does not work anymore. great job. Using it now on my android. However am i right that this github project is the old version and the new foobar link app is not currently open source?

**Reply by deckerth** [reply]
The current branch ist Query API. Can you access IT?

https://github.com/deckerth/FoobarRemoteController2/tree/QueryAPI

**Reply by eikaramba** [reply]
ah nice, thank you. yes i can see it.

**Reply by deckerth** [reply]
Wir nutzen ab sofort den Query Endpunkt. Das wird den Akkuverbrauch deutlich senken. :)

**Comment by radgroza** [comment]
Hi, I have a problems with using the Link app and Browsing capability. Yes, I can browse the directory structure, it is all fine... but I cannot add anything from the browser to a playlist. I don't see a little Plus icon to the right of items in the browser. I see only a static "check" icon, but it does not do anything. The entire Browser seems like dead feature. It allows to view stuff but nothing else. Can it be fixed somehow?

**Reply by deckerth** [reply]
I was partly able to reproduce the issue. However, I am able to set checkboxes like this:
![Screenshot_20260320-080906.png](https://github.com/user-attachments/assets/753c2825-aed0-4611-8d25-0cecf00fd233)

It is not possible to deselect a checkbox after it was selected though. Also when restarting the browser, the checkboxes are still set. These are two bugs that we will fix. A new version is planned by end of March.

After leaving the browser, the selected tracks are added to the playlist, which is still working as designed.

**Reply by deckerth** [reply]
The new version fixed the issue with selections not being reset when the browser is started. It is by design that checkboxes cannot be deselected.
But I think that all this did not solve your problem, right?
